modules/nf-core/bbmap/filterbyname/main.nf

```diff
@@ -0,0 +1,59 @@
+process BBMAP_FILTERBYNAME {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bbmap=39.01" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.01--h311275f_0':
+        'quay.io/biocontainers/bbmap:39.01--h311275f_0' }"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*.$output_extension"), emit: reads
+    tuple val(meta), path('*.log')                , emit: log
+    path "versions.yml"                           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args               = task.ext.args   ?: [ args:"" ]
+    args.args = args.args ? "$args.args" : ""
```

Comment:
```suggestion
    def args               = task.ext.args   ?: ""
```

Reply:
if args doesn't exists it will be evaluated anyways to an empty string, no?

Reply:
the arguments itself are defined as `args.args` (that's why it's parsed that way)